Library/Homebrew/rubocops/cask/array_alphabetization.rb

```diff
@@ -0,0 +1,73 @@
+# typed: true
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Cask
+      class ArrayAlphabetization < Base
+        extend AutoCorrector
+
+        def on_send(node)
+          return unless [:zap, :uninstall].include?(name = node.method_name)
+
+          node.each_descendant(:pair).each do |pair|
+            symbols = pair.children.select(&:sym_type?).map(&:value)
+            # For `zap`s, we only care about `trash` arrays.
```

Comment:
Does it make sense to treat `zap` and `uninstall` differently? Under the hood they use the same code.